docs/guide/cookbook/module.md

```diff
@@ -831,3 +831,39 @@ Inside `check` we tell the filter to just be applied if the attribute is named e
 Inside `filter` we extend the Elasticsearch query-chain by our desired filters, using the `bodybuilder` library syntax.
 
 That's it, now we are able to filter by a complex query in only one line inside VSF.
+
+## 7. Working with translations in module
+
+Translations are provided in `core/i18n/resource/i18n/{your-locale}.csv` file and can be extended or overridden in `src/modules/{my-module}/resource/i18n/{your-locale}.csv` accordingly.
+
+Remember, translations from `src/module/{my-module}/resource/i18n` are overridden by translation from `src/themes/{my-theme}/resource/i18n`.
```

Comment:
Let's merge this sentence and one above. Let's make it clear that theme's translations have bigger priority than ones from modules.